x-pack/plugin/ent-search/qa/rest/src/yamlRestTest/resources/rest-api-spec/test/entsearch/20_put.yml

```diff
@@ -0,0 +1,72 @@
+setup:
```

Comment:
nit: Let's call the file 20_engine.yml or  20_engine_basic to leave room for testing the get and remove method?